lib/internal/Magento/Framework/Encryption/Crypt.php

```diff
@@ -10,6 +10,7 @@
  * Class encapsulates cryptographic algorithm
  *
  * @api
+ * @deprecated
```

Comment:
Please add `@see` with the reference to the replacement.

Reply:
Please add the class to the static test that searches for obsolete code, so nobody tries to use it.